soc/arm/microchip_mec/mec172x/Kconfig.defconfig.mec172xnsz

```diff
@@ -39,4 +39,8 @@ config SPI_XEC_QMSPI_LDMA
 	default y
 	depends on SPI
 
+config KSCAN_XEC
+	default y
+	depends on KSCAN
```

Comment:
driver can default based on compatible status instead (in driver's Kconfig):
```
default $(dt_compat_enabled,$(DT_COMPAT_XXXX))
```

Reply:
updated accordingly